src/toast/mpi.py

```diff
@@ -114,7 +114,7 @@ class Comm(object):
     """Class which represents a two-level hierarchy of MPI communicators.
 
     A Comm object splits the full set of processes into groups of size
-    "group".  If group_size does not divide evenly into the size of the given
+    `groupsize`.  If groupsize does not divide evenly into the size of the given
     communicator, then those processes remain idle.
```

Comment:
I know you didn't change this line but it should be `some processes`, not `those processes`.